linchpin/examples/controller-provisioning/ks.kstest-controller.cfg

```diff
@@ -0,0 +1,15 @@
+url --url="https://dl.fedoraproject.org/pub/fedora/linux/releases/29/Everything/x86_64/os/"
+lang en_US.UTF-8
+keyboard --vckeymap=us --xlayouts='us'
+rootpw --plaintext anaconda
+timezone --utc Europe/Prague
+clearpart --all
+
+autopart
+
+%packages --ignoremissing
+@core
+@base
+%end
+
+sshkey --username root "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDJjCafb/FZ4hIuQXfUPvhkZmQT2na//fl2rmnG5i65EyJXrWZyO2VM4n4rDmYT2scxp3BDV1ovacTetiqUW3cBID+pzuyVmRonsE6eNudlP47BWv14kPY3SocImt4lgjA2bag4yhgN7mx20c3bLGL7L25D27SZwDYfBbPUl1uuypmMlyiLZ7Cwpf+qEeobfk5nwjZMDZEL9abPk2HYIm+2udLCqUtVj/51lQ+XL5AH1EJvMdh4Fv6GuqV69jSZoc9XcPoUX22NXIirym0Tpzro/j4xOZ0jDN5E012fdMcKwFwLMiXrSq1Amgk4B7z0EQQ4z3/by/EAsxCEkyuoa5vr Generated-by-Nova"
```

Comment:
What is this key, is it real key used somewhere?

Reply:
It is kstests public key I am using. I can put there any other generated public key, but I don't think it matters.

Reply:
I guess public key is fine.